scripts/amara_udpdate_captions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+if (file_exists(dirname(__FILE__).'/../config.inc.php')) {
+    require_once dirname(__FILE__).'/../config.inc.php';
+} else {
+    require_once dirname(__FILE__).'/../config.sample.php';
+}
+
+//Establish mediahub connection
+$media_hub = new UNL_MediaHub();
+
+$db = Doctrine_Manager::getInstance()->getCurrentConnection();
+
+//Get all orders that have not been completed.
+$media_list = new UNL_MediaHub_MediaList(array('limit'=>10000));
```

Comment:
I suggest putting magic numbers in a place where they can be quickly changed without having to scan through long lines of code.